### PR TITLE
fix: clean up UI

### PR DESCRIPTION
**Pull request by DanHuy77**

### CODE REVIEW

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css [hunk 1]
```diff
@@ -1,30 +1,27 @@
...
     flex-direction: column;
 }
+
 .jr-header {
     display: flex;
     align-items: center;
     height: 20%;
 }
+
 .jr-btn-back {
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css:25]: Remove empty line

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css [hunk 2]
```diff
@@ -40,12 +37,14 @@ body {
...
+
 .jr-img-logo {
- 
+
     height: 100px;
     width: 125px;
     margin: 0 auto;
 }
+
 .jr-btn-sound {
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css]: Remove empty line

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css [hunk 3]
```diff
@@ -40,12 +37,14 @@ body {
     color: #000000;
     justify-content: center;
 }
+
 .jr-img-logo {
- 
+
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css]: Remove empty line

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css [hunk 4]
```diff
@@ -60,6 +59,7 @@ body {
     background-color: #C6A88F;
 
 }
+
 .jr-center {
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css]: Remove empty line

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css [hunk 5]
```diff
@@ -143,38 +152,44 @@ option {
...
     display: flex;
     flex-direction: column;
     width: 64%;
     border-radius: 10px;
-    margin: 8px ;
+    margin: 8px;
     height: 95%;
 
-    
+
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css]: Remove empty lines

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css [hunk 6]
```diff
@@ -143,38 +152,44 @@ option {
...
 }
+
 .jr-setting {
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css]: Remove empty line

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css [hunk 7]
```diff
@@ -184,18 +199,21 @@ span {
...
     height: 25%;
 }
+
 .jr-column {
     flex: 1;
     font-weight: bold;
     font-size: 16px;
 }
+
 .jr-data {
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css]: Remove empty line

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css [hunk 8]
```diff
@@ -184,18 +199,21 @@ span {
...
 
     display: flex;
     flex-wrap: wrap;
     overflow: auto;
     height: 85%;
 }
+
 .jr-detail {
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css]: Remove empty line

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css [hunk 9]
```diff
@@ -205,28 +223,33 @@ span {
     border-radius: 10px;
     margin: 6px;
 }
+
 .jr-detail-setting {
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css]: Remove empty line

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css [hunk 10]
```diff
@@ -205,28 +223,33 @@ span {
...
 
     display: flex;
     flex-direction: column;
     height: 100%;
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css]: Remove empty line

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css [hunk 11]
```diff
@@ -205,28 +223,33 @@ span {
...
     display: flex;
     justify-content: center;
 }
+
 .jr-img-setting {
     height: 80%;
     width: auto;
 }
+
 .jr-action {
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css]: Remove empty line

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css [hunk 12]
```diff
@@ -251,4 +275,13 @@ span {
     font-weight: bold;
     text-align: center;
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css]: Remove empty line

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js [hunk 13]
```diff
@@ -184,15 +176,18 @@ const JoinRoom = () => {
           <div className="col-7 section">
             <div className="row h-80"></div>
             {role == 1 && (
-              <div className="row h-25 align">
-                <div className="col-6 right">
-                  <button className="button" onClick={handleInviteClick}>
+              <div className="row h-25 align d-flex">
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js]: align class already has display: flex iirc

File: the-telephone-app-fe/src/components/ShowResult/ShowResult.js [hunk 14]
```diff
@@ -223,23 +223,36 @@ const ShowResult = () => {
             {timer - 1 >= resultSet.length * 4 ? (
               <div className="mt-4">
                 {role === 1 && (
-                  <div className="row h-20 align">
-                    <div className="col-6 right">
+                  <div className="row h-20 align d-flex">
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/ShowResult/ShowResult.js]: align class already has display: flex iirc

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css (hunk 12 above)
Comment (ces-manhnguyen) [reply the-telephone-app-fe/src/components/JoinRoom/JoinRoom.css]: You missed this.

**Review by ces-manhnguyen — APPROVED** [review]

**Review by thoa01 — APPROVED** [review]

**Pull request merged**